lib/march_hare/session.rb

```diff
@@ -608,6 +613,32 @@ def tls_certificate_password_from(opts)
       self.class.tls_certificate_password_from(opts)
     end
 
+    # @private
+    def init_default_exception_handler(logger)
+      StrictExceptionHandler.new(logger)
```

Comment:
AFAICS this is also the default in the java lib
https://github.com/rabbitmq/rabbitmq-java-client/blob/master/src/main/java/com/rabbitmq/client/impl/DefaultExceptionHandler.java